packages/flame/lib/src/parallax.dart

```diff
@@ -195,13 +195,13 @@ class ParallaxAnimation extends ParallaxRenderer {
 
     final animation =
         await SpriteAnimation.load(path, animationData, images: images);
-    final prerendedFrames = await Future.wait(
+    final preRendedFrames = await Future.wait(
```

Comment:
This should be the same as in the other parallax class

Reply:
sorry, forget that 
will fix that